src/components/home/modals/AddToken.vue

```diff
@@ -111,6 +111,7 @@ export default {
             if (tmpUserInfo && tmpUserInfo.tokens) {
                 tokens = JSON.parse(tmpUserInfo.tokens)
             }
+            this.tokenId = this.tokenId.replace(/\s*/g, '')
```

Comment:
We'd better not remove white space here. Use `trim` method in `getTokenId()` function.